### PR TITLE
fixes #124

### LINKED ISSUE #124: Rank by most solves ever
See https://groups.google.com/d/msg/wca-software/62zHEzLCm1Y/zdsgvTSBYyYJ, which discusses Sebastien's 10k solve milestone.

### PR DESCRIPTION
**Pull request by SAuroux**
That looked like an easy thing to do for me. Given that I am kind of the reason for this being an issue, I gave it a try. :)

### CODE REVIEW

File: webroot/results/includes/statistics/most_solves_in_one_competition_or_year.php [hunk 1]
```diff
@@ -1,5 +1,24 @@
 <?php
 
+$totalsolves = polishMostSolvesAttempts( "
+  SELECT    personId,
+            competitionId whereId,
```
Comment (jfly) [inline webroot/results/includes/statistics/most_solves_in_one_competition_or_year.php]: What does `competitionId` mean in this context?

Comment (jfly) [inline webroot/results/includes/statistics/most_solves_in_one_competition_or_year.php]: Not your fault, but the the variable names in here make no sense to me. There's `$totalsolves`, `$solves`, and `$attempts`. From reading, it looks like all 3 of these variables include both a number of solves *and* a number of a attempts, it's just either by career, competition, or year. If I'm understanding this correctly, would you change the three variables to something like `$mostSolvesInACareer`, `$mostSolvesInACompetition`, `$mostSolvesInAYear`?

**Comment by jfly** [comment]
Cool stuff, @SAuroux! It looks like the filename `most_solves_in_one_competition_or_year` is no longer accurate, would you please fix that? Perhaps `most_solves_in_one_competition_or_year_or_career`, or just plain `most_solves`...

**Comment by jfly** [comment]
Here's what I see with your latest changes:

2015-09-30_1451x366_kaladin

**Comment by SAuroux** [comment]
Ok, sorry for my messy attempt then, please refuse the PR.

I will make sure to have a local testing environment available for future
code changes.

2015-09-30 9:46 GMT+02:00 Jeremy Fleischman <notifications@github.com>:

**Comment by timhabermaas** [comment]
@SAuroux The vagrant thing described in the README works pretty well for me (on Mac OS X). No need to install PHP or MySQL.

**Comment by jfly** [comment]
OK! Thanks for trying, @sauroux.

**Pull request closed without merging**